alerting/src/main/kotlin/org/opensearch/alerting/util/IndexUtils.kt

```diff
@@ -155,5 +156,20 @@ class IndexUtils {
 
             return result
         }
+
+        @JvmStatic
+        fun getWriteIndexNameForAlias(clusterService: ClusterService, alias: String): String? {
+            return clusterService.state().metadata().indicesLookup?.get(alias)?.writeIndex?.index?.name
+        }
+
+        @JvmStatic
+        fun isIndexWriteIndex(clusterService: ClusterService, index: String): Boolean {
+            clusterService.state().metadata().indicesLookup.values.forEach {
```

Comment:
This is checking if concrete index is writeIndex of some alias. Since we support passing custom queryIndex name, we can't just take predefined queryIndex and check it.